django_dsg/settings.py

```diff
@@ -141,7 +141,7 @@
         'rest_framework.throttling.UserRateThrottle',
     ],
     'DEFAULT_THROTTLE_RATES': {
-        'anon': '100000000/hour',
-        'user': '99999991/minute',
+        'anon': '1000/hour',
```

Comment:
What if we become a super used game, like among us?

Reply:
We would need to implement admin settings. Would go with the spring where we implement log-in.